MigrationV3V4/CSharp/Skills/V4SimpleRootBot/AdapterWithErrorHandler.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Builder.Integration.AspNet.Core.Skills;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.BotBuilderSamples.SimpleRootBot.Bots;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+
+namespace Microsoft.BotBuilderSamples.SimpleRootBot
+{
+    public class AdapterWithErrorHandler : BotFrameworkHttpAdapter
```

Comment:
>AdapterWithErrorHandler [](start = 17, length = 23)

is it worth highlighting this as an important top level class?

Reply:
Added a comment explaining the purpose of this class.  Thanks!